### ISSUE TITLE
Header logo and header background image disappear on fragment

### ISSUE THREAD

**Issue by RodrigezJson** [florent37/MaterialViewPager#153]
Hello guys,

I need you help.My header logo and header background image disappear if i changed to the another fragment.This is happen when i scroll down the card view on fragment A without scrolling up and then i change to the fragment B. 

Fragment A
![screenshot_2015-09-01-10-40-00](https://cloud.githubusercontent.com/assets/8197462/9595133/87f3bd8e-5097-11e5-97ce-1d66e1afd895.png)

Scroll down fragment A without scroll it up back. Then i go to fragment B using navigation drawer.
![screenshot_2015-09-01-10-40-07](https://cloud.githubusercontent.com/assets/8197462/9595136/8f7fca84-5097-11e5-846f-5197311f3295.png)

Header logo and header background image disappear on fragment B
![screenshot_2015-09-01-10-43-13](https://cloud.githubusercontent.com/assets/8197462/9595144/9f071dea-5097-11e5-8ba4-22a898e19dcf.png)

Please help me to figure out on this problem. This is cause me pain a few days. :)
Thank You

**Comment by florent37** [comment]
Can you try with the last update please ?

**Comment by tolawuwo** [comment]
Yes I am also having the same issue with the latest code. Could you send a sample of how to implement my own header. For example what would the layout look like. The fromurlcolor is not pointing to the images on the header

**Comment by chnouman** [comment]
same error plz tell me how to fix
